05_concurrency/homework/main.go

```diff
@@ -0,0 +1,163 @@
+package main
```

Comment:
Great work! But in future you should consider using sync.RWMutex for performance optimization.